### PR TITLE
Removed mobile-list.twig now just using list.twig

### PR DESCRIPTION
**Pull request by erhanask**

### CODE REVIEW

File: resources/views/list/partials/thead.twig [hunk 1]
```diff
@@ -1,39 +1,39 @@
-<thead>
-    <tr class="text-white">
-        <th class="text-nowrap" colspan="2">{{ trans('visiosoft.module.advs::field.ad_title') }}</th>
+<thead class="">
```
Comment (fatihalp) [inline resources/views/list/partials/thead.twig:1]: empty class @erhanask

File: resources/views/list/partials/thead.twig [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
-        {% endif %}
-        <th class="text-nowrap sortable{{ param.sort_by == 'sort_price_up' ? ' sort-by sort-desc' : (param.sort_by == 'sort_price_down' ? ' sort-by') }}"
-            data-sort-asc="sort_price_down" data-sort-desc="sort_price_up">
-            {{ trans('visiosoft.module.advs::field.price.name') }}
+            {% endif %}
+        </th>
+    {% endfor %}
+
+    {% if listLocation %}
+        <th class="text-nowrap sortable{{ param.sort_by == 'address_z_a' ? ' sort-by sort-desc' : (param.sort_by == 'address_a_z' ? ' sort-by') }}"
```
Comment (fatihalp) [inline resources/views/list/partials/thead.twig:17]: is it necessary? @erhanask

Comment (erhanask) [reply resources/views/list/partials/thead.twig:17]: For desktop these are can be necessary, so i just added "d-none" class on mobile for <tr> tag that used inside of <thead> tag . @fatihalp

File: resources/config/settings/settings.php [hunk 3]
```diff
@@ -586,6 +586,12 @@
         'config' => [
             'default_value' => false
         ]
+    ],
+    'show_gifs_listing_main_pages' => [
+        'type' => 'anomaly.field_type.boolean',
+        'config' => [
+            'default_value' => false
```
Comment (fatihalp) [inline resources/config/settings/settings.php:593]: could be true by default we want it

**Review by fatihalp — APPROVED** [review]

**Review by spektra2147 — APPROVED** [review]

**Pull request merged**